nRF9160Master/src/PacketHandler/PacketHandler.c

```diff
@@ -77,6 +79,17 @@ bool ParsePacket(uint8_t *pucRcvdBuffer, _sPacket *psPacket)
     return bRetVal;
 }
 
+int SetBleStatus (bool flag)
+{
+    bool BleStatusFlag = flag;
+}
+
```

Comment:
I assume that you are having function headers.  If not please add them, it will be easily understandable for anyone who checks code like the purpose of function, arguments to the function, what it returns.. etc. Please follow the same syntax which other functions have now.